testing/ef_tests/check_all_files_accessed.py

```diff
@@ -27,20 +27,6 @@
     "tests/.*/.*/ssz_static/PowBlock/",
     # light_client
     "tests/.*/.*/light_client",
-    # LightClientStore
-    "tests/.*/.*/ssz_static/LightClientStore",
-    # LightClientUpdate
-    "tests/.*/.*/ssz_static/LightClientUpdate",
-    # LightClientSnapshot
-    "tests/.*/.*/ssz_static/LightClientSnapshot",
-    # LightClientBootstrap
-    "tests/.*/.*/ssz_static/LightClientBootstrap",
-    # LightClientOptimistic
-    "tests/.*/.*/ssz_static/LightClientOptimistic",
-    # LightClientFinalityUpdate
-    "tests/.*/.*/ssz_static/LightClientFinalityUpdate",
-    # LightClientHeader
-    "tests/.*/.*/ssz_static/LightClientHeader",
```

Comment:
The tests are failing on CI because it hasn't accessed the capella & deneb files for light client types, might have to wait until we have the types added.